.github/workflows/dev-workflow.yml

```
@@ -188,4 +188,4 @@ jobs:
         with:
           base: main
           head: dev
-          token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)